rotkehlchen/tests/utils/dataimport.py

```diff
@@ -697,133 +698,110 @@ def assert_nexo_results(rotki: Rotkehlchen):
     warnings = rotki.msg_aggregator.consume_warnings()
     errors = rotki.msg_aggregator.consume_errors()
     assert len(errors) == 0
-    assert len(warnings) == 0
+    assert len(warnings) == 6
```

Comment:
What are these 6 warnings about? I would adjust our logic to have 0 warnings or at least check the warnings' content that we get here, what do you think?

Reply:
Does this mean these transactions are completely ignored? Can we adjust to support them?

Reply:
The entries are completely ignored, yeah.
https://github.com/rotki/rotki/blob/develop/rotkehlchen/data_import/importers/nexo.py#L42

Reply:
Seems like `ignored_entries` are not the case, are they? If transaction type is mentioned in `ignored_entries`, it is skipped and no warning is raised, or am I misunderstanding something?

Reply:
Yeah, you're right. I was the one mixing things together.